src/global/styles/typography.scss

```diff
@@ -51,3 +51,10 @@ html {
 .sbb-legend {
   @include typo.legend;
 }
+
+// In smaller title font-sizes, the space after the title is smaller than the default paragraph space before.
+// Due to margin collapsing, the paragraph space wrongly wins.
+// To prevent the too large gap, we reset the paragraph space.
```

Comment:
```suggestion
// Due to margin collapsing, the wrong paragraph space wins.
// To prevent the mistakenly large gap, we reset the paragraph space.
```